src/main.rs

```diff
@@ -2860,14 +2860,16 @@ fn run_steps(ib: &mut ImageBuilder) -> Result<()> {
                 struct SeedSmfArgs {
                     debug: Option<bool>,
                     apply_site: Option<bool>,
+                    seed: Option<String>,
                 }
 
                 let a: SeedSmfArgs = step.args()?;
                 let debug = a.debug.unwrap_or(false);
                 let apply_site = a.apply_site.unwrap_or(false);
+                let seed = a.seed.unwrap_or("global".to_string());
```

Comment:
```suggestion
                let seed = a.seed.as_deref().unwrap_or("global");
```

I think this is probably slightly preferable.  `seed` will be a `&str`, then, so you can remove the `&` on 2872.

Reply:
Done.